net.sourceforge.kmetronome.json

```diff
@@ -0,0 +1,79 @@
+{
+  "app-id": "net.sourceforge.kmetronome",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-21.08",
+  "sdk": "org.kde.Sdk",
+  "command": "kmetronome",
+  "rename-icon": "kmetronome",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
+    /* MIDI */
+    "--device=all",
+    /* Allow loading, saving files from anywhere (portals don’t work yet) */
+    "--filesystem=home",
+    /* Allow other instances to see lockfiles */
+    "--env=TMPDIR=/var/tmp",
+    /* On SuSE it seems to be necessary for sound to work */
+    "--env=ALSA_CONFIG_PATH="
+  ],
+  "cleanup": [
+    "/share/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "*.la"
+  ],
+  "modules": [
+    {
+      "name": "drumstick",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DBUILD_ALSA=ON",
+        "-DBUILD_FILE=OFF",
+        "-DBUILD_RT=OFF",
+        "-DBUILD_WIDGETS=OFF",
+        "-DBUILD_DOCS=OFF",
+        "-DBUILD_UTILS=OFF",
+        "-DBUILD_TESTING=OFF"
+      ],
+      "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/cmake",
+        "/lib/plugins",
+        "/lib/pkgconfig",
+        "/share/mime",
+        "/share/icons/hicolor/*/apps",
+        "*.desktop",
+        "/lib/*.so"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/drumstick/2.5.0/drumstick-2.5.0.tar.bz2",
+          "sha256": "8709d8a7c438c1d09091827ce5fd8286d387a046a8f717a1a34546d9ca3d902a"
+        },
+        {
+          "type": "patch",
+          "path": "install-header-macros.patch"
+        }
+      ]
+    },
+    {
+      "name": "kmetronome",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [ "-DBUILD_DOCS=OFF" ],
+      "post-install": [
+        "install -Dm644 -t /app/share/metainfo/ net.sourceforge.kmetronome.appdata.xml"
```

Comment:
This is not installed by upstream? (it should be in a future release)

Reply:
Yes, a small mistake on my part. It is included in the SVN now: 
https://sourceforge.net/p/drumstick/code/623/

Reply:
I'm following here the same post-install step of VMPK and dmidiplayer. Do you mean that it is wrong? Or that I should remove this step entirely?